modules/core/04-channel/types/channel_test.go

```diff
@@ -85,38 +85,38 @@ func TestSubsetOf(t *testing.T) {
 		},
 		{
 			"unordered -> ordered",
```

Comment:
think we need to change the names of the test cases to align with the new order of arguments

Reply:
Doesn't it make sense like this? We're going from "unordered -> ordered" so the parentOrder is UNORDERED and we're checking if ORDERED is a valid subset of, which should be false.